### PR TITLE
add eslint to the repo.

### LINKED ISSUE #3: Add eslint

### PR DESCRIPTION
**Pull request by Abhirup-99**
Fixes #3.

### CODE REVIEW

**Comment by Abhirup-99** [comment]
Fixes #3.

File: src/App.jsx [hunk 1]
```diff
@@ -5,24 +5,26 @@ const EosForm = require("./EosForm");
...
+  const [searchGif, updateSearchGif] = React.useState(<div />);
 
   const onSearch = () => {
     const svgName = refContainer.current.value;
     const request = new XMLHttpRequest();
     const url = `https://cdn.jsdelivr.net/npm/eos-icons@latest/svg/${svgName}.svg`;
     request.open("GET", url);
-    updateSearchGif(<img src={Spinner} width="40px" height="40px" />);
+    updateSearchGif(
+      <img src={Spinner} width="40px" height="40px" alt="Loading" />
```
Comment (kbhutani0001) [inline src/App.jsx:19]: is spinner a loading icon? if yes, lets try to use one of our own and not a gif

File: src/EosForm.jsx [hunk 2]
```diff
@@ -1,44 +1,67 @@
 const React = require("react");
-const styles = require("./dialog.css");
```
Comment (kbhutani0001) [inline src/EosForm.jsx:2]: can we try the es6 imports? so that it is consistent across all our code base

**Comment by kbhutani0001** [comment]
do we have any design or anything to know what we are building and how it will look @Abhirup-99 ?

**Comment by Abhirup-99** [comment]
Hi, thanks for the review. I was a bit busy with my college work. Updating shortly.

**Comment by Abhirup-99** [comment]
latest ui.

https://user-images.githubusercontent.com/48902681/122679407-599d8980-d208-11eb-9fd8-d996de6e0013.mp4

**Review by kbhutani0001 — APPROVED** [review]

**Pull request merged**